templates/transcript.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Template for transcripts.
+ * Intentionally barebones with the minimum html for use by tools.
+ *
+ * @package tenup_podcasting
+ */
+
+?>
+<!DOCTYPE html>
+<html lang="<?php echo esc_attr( get_locale() ); ?>">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title><?php esc_html_e( 'Transcript', 'simple-podcasting' ); ?> - <?php echo esc_html( get_the_title() ); ?></title>
```

Comment:
This is better for internationalization. 

I've removed  the escaping for `get_the_title()` as it's escaped on save and considered safe.

```suggestion
	<title><?php
	printf(
		esc_html__( 'Transcript - %s', 'simple-podcasting' ),
		get_the_title()
	);
	?></title>
```

Reply:
@peterwilsoncc, Most of the data is sanitised/escaped on save, and this is a very small concern, but in case an attacker would get hold of DB or is able to modify values of wp_posts, I think an XSS attack would be easy to do here, and I don't see much of an issue with escaping here(unless we're facing double escaping issues). So I'm in favour of keeping the escape, but let me know what you think about it.